lib/src/config/routes/genres_list_route.dart

```diff
@@ -0,0 +1,47 @@
+import 'package:flutter/material.dart';
+
+import '../../core/util/constants.dart' as constants;
+import '../../domain/entity/genre.dart';
+import '../../presentation/view/genre_movie_list.dart';
+
+Route genreListRoute(Genre genre){
```

Comment:
You are making almost the same functionality that you have on expanded_list_route, could be better approach to have only one RouteGenerator, because the only thing that is changing is the `PageBuilder` Widget